website/utils/renderer.ts

```diff
@@ -7,7 +9,15 @@ export const renderStatic = async (callback: () => string) => {
     throw new Error('did you forget to return html from renderToString?');
   }
 
-  const { ids, css } = extractCritical(html);
+  const { extractCritical: websiteExtractCritical } = createEmotionServer(
+    cache,
+  );
+
+  const { ids: websiteIds, css: websiteCss } = websiteExtractCritical(html);
+  const { ids: lgIds, css: lgCss } = lgExtractCritical(html);
+
+  const ids = [...websiteIds, ...lgIds];
+  const css = websiteCss + lgCss;
```

Comment:
could we handle this concatenation with a bit more paranoia? just not that comfortable with css formed like `.foo{...}.bar{...}`
```suggestion
  const css = `${websiteCss}\n${lgCss}`;
```

Reply:
I actually don't think this is necessary to separate them out. Was console-logging the websiteCss and wasn't seeing anything separate. Removed this entirely, but added a comment to the extractCritical function used to extract the styles from LGUI Emotion Server.